40-car-detector/README.md

```diff
@@ -0,0 +1,229 @@
+
+#### Environment Variables
+
+There are 2 [environment variables](https://dev.blues.io/guides-and-tutorials/notecard-guides/understanding-environment-variables/) you may want to configure:
+
+- `confidence_threshold`: This is the probability threshold mentioned in the previous section. It should be greater than 0 and less than 1. By default, it's set to 0.95.
+- `publish_period`: This is how often `events.qo` notes are synced to Notehub, in seconds. By default, it's set to 300 (5 minutes).
+
+To change these values, go to your Notehub project and click Environment in the left-hand menu. Under Project environment variables, enter your desired values and click Save:
+
+![Setting environment variables](./assets/env_vars.png)
+
+The next time data is synced to the Notecard, the environment variables will update on the MCU.
+
+#### Running `main.py`
+
+Before running `main.py`, uncomment this line: `# product_uid = 'com.your-company:your-product-name'`. Replace `com.my-company.my-name:my-project` with the [ProductUID of the Notehub project](https://dev.blues.io/notehub/notehub-walkthrough/#finding-a-productuid) you created in [Notehub Setup](#notehub-setup).
+
+At this point, you could copy `main.py` onto the MCU, reboot the MCU, and the code would start running. However, you won't be able to see the detections (or mis-detections). Instead, use OpenMV IDE to run the code.
+
+1. Open OpenMV IDE.
+2. Click the Open File icon in the left-hand toolbar and select `main.py`.
+3. Click the Connect button in the bottom left corner.
+4. Click the Start button below the Connect button.
+
+The code is now running on the MCU, and you'll be able to see what the camera sees in the Frame Buffer. Additionally, when the model detects a car, the code will draw a circle at the point of the detection in the Frame Buffer.
+
+This is super valuable for evaluating the performance of the model in real time. Additionally, you can monitor serial logs by clicking Serial Terminal at the bottom of the IDE window.
+
+Once you're happy with how the model is performing, you can copy `main.py` over to the board so that it'll run outside the IDE context with this command:
+
+```
+python pyboard.py -d <serial port> --no-soft-reset -f cp main.py :/
+```
+
+Note that for `pyboard.py` to work, you'll need to install [pyserial](https://pypi.org/project/pyserial/) with `pip install pyserial`, if you don't have it installed already. Make sure to replace `<serial port>` with your serial port. Unplug the camera board and plug it back in to reboot the device. `main.py` will start running after boot up.
```

Comment:
Could you be more specific about what should be unplugged? I'm guessing the USB cable,  but saves guessing if we call it out explicitly.

I'm also imagining the development PC is no longer required, so it's worth calling out that the development PC is only required while training the model.